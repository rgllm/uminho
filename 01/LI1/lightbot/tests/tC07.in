["Caac","caac","caac","CccC","3 3 S","AAALDAAALDAAAL"]