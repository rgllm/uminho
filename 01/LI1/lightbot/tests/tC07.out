["3 0","0 0","0 3","FIM 14"]